src/app/App.vue

```diff
@@ -1,13 +1,38 @@
 <template>
-	<div class="container max-width-1200 mx-auto">
-		<h1 class="text-center font-bold text-xl">App</h1>
+	<div class="container mx-auto">
+		<h1 class="text-center text-xl my-5 font-hairline">Here you can find some comics info!</h1>
+		<SearchBar />
+		<div v-if="areFetching">
+			<Loader />
+		</div>
+		<div v-else-if="allComicses && allComicses.length">
```

Comment:
Let's call it more generic, like `searchResult`. If in future we will want to combine searches with not only by comics, we won't be forced to refactor this name